File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/localevent/ArenaPlayerLeaveLocalEvent.java
package com.gmail.borlandlp.minigamesdtools.arena.localevent;

import org.bukkit.entity.Player;

@ArenaEventHandler
public class ArenaPlayerLeaveLocalEvent extends ArenaEvent {
    private Player player;

    public ArenaPlayerLeaveLocalEvent(Player player) {
        this.player = player;
    }

    public Player getPlayer() {
        return player;
    }

    public String toString() {
        return "{Event:" + this.getClass().getSimpleName() + " Player" + this.getPlayer() + "}";
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/localevent/ArenaPlayerLeaveLocalEvent.kt
package com.gmail.borlandlp.minigamesdtools.arena.localevent

import org.bukkit.entity.Player

@ArenaEventHandler
class ArenaPlayerLeaveLocalEvent(val player: Player) : ArenaEvent() {
    override fun toString(): String {
        return "{Event:" + this.javaClass.simpleName + " Player" + player + "}"
    }
}